### ISSUE TITLE
What are the compatible cameras for this setup ?

### ISSUE THREAD

**Issue by chanakaDe** [kellyschrock/3drsolo_xiaomi#1]
This solution is great buddy. Thanks for sharing with us.
Actually this is not an issue. Just need to confirm. There are some new models available in YI action camera series. Is there any specific camera for this hack ? Or can we use any YI platform camera ?

And also, after we have done this, can we rest out action camera to work as earlier ? (I mean default setup)

**Comment by kellyschrock** [comment]
I haven't looked at some of the new Yi cameras TBH, but it would be nice to
support them if possible.

**Comment by chanakaDe** [comment]
@kellyschrock  Can you please tell me what's the camera model you have tried ? Sorry if I'm disturbing buddy.

**Comment by kellyschrock** [comment]
I had a Xiaomi Yi, from 2014 or thereabouts. I understand they've been
upgraded considerably since then.

On Wed, Nov 29, 2017 at 11:21 AM, Chanaka De Silva <notifications@github.com

**Comment by chanakaDe** [comment]
@kellyschrock  Hi buddy,
I was looking at all the posts about this setup. http://diydrones.com/group/solo/forum/topics/controlling-a-xiaomi-yi-camera-from-solo
Have a question. I need a camera on my Solo only for FPV purpose.  I have Alfa antennas and i'm going more than 1 mile range and need to see where my bird is. :1st_place_medal:  Because I love Solo a lot. Anyway this is not my drone for photography. So no need to control camera over remote or still shots. Can i just connect camera with Solo and take FPV ? Without these hard config ? Is it possible ? Ports and all are ok ?

**Comment by kellyschrock** [comment]
Yes, you should be able to just plug the HDMI cable into the Yi and see its
output on the app with no problem. Any camera with an HDMI port should work
in that case.


On Thu, Dec 28, 2017 at 10:30 AM, Chanaka De Silva <notifications@github.com
